.github/workflows/vscode-versions-matrix.yml

```diff
@@ -12,7 +12,7 @@ jobs:
     runs-on: ubuntu-20.04
     strategy:
       matrix:
-        vscode-version: ["1.61.2", "1.66.0"]
+        vscode-version: ["1.65.2", "1.66.2"]
```

Comment:
Do we really need to upgrade the version?
it might cause troubles with Theia/Che

Reply:
ok thanks for explanation! 👍 

I agree we should keep 1.61.2 I will change that back, thanks for catching that!